build/cmake/GenerateUninstaller.cmake

```
@@ -298,4 +274,10 @@ foreach(dir IN LISTS dirs_to_remove)
     add_rmdir("${relpath}")
 endforeach()
 
+# Allow the batch script delete itself without error.
+if(WIN32)
+    append_line("  true")
```

```suggestion
    append_line("  echo Uninstall complete")
```

To fix observed error:
```
'true' is not recognized as an internal or external command,
operable program or batch file.
```

---

Looks like I was inadvertently depending on Cygwin (EVG) and MSYS2 (local) features which are not supported by pure CMD. Replaced with a call to `ver>nul` (see [here](https://stackoverflow.com/a/1113735) and [here](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/ver)).